pkg/kubecfg/init.go

```diff
@@ -24,5 +27,8 @@ func NewInitCmd(name string, rootPath metadata.AbsPath, specFlag string, serverU
 
 func (c *InitCmd) Run() error {
 	_, err := metadata.Init(c.name, c.rootPath, c.spec, c.serverURI, c.namespace)
+	if err == nil {
+		log.Info("ksonnet app successfully created! Next, try creating a component with `ks generate`")
```

Comment:
nice! I like this. 
nit: period at the end of this.

Reply:
I'll add a period here. 

Note: Not sure why stylistically we haven't been adding punctuation to the end of our logging messages -- we might want to go through this later...